bin/check-cloudwatch-metric.rb

```diff
@@ -91,11 +91,11 @@ class CloudWatchMetricCheck < Sensu::Plugin::Check::CLI
   option :compare,
          description: 'Comparision operator for threshold: equal, not, greater, less',
          short: '-o OPERATION',
-         long: '--opertor OPERATION',
+         long: '--operator OPERATION',
          default: 'greater'
 
   option :no_data_ok,
-         short: '-n',
+         short: '-O',
```

Comment:
This feels less obvious to me as to why this should change is there another cloudwatch check that uses `-O` rather than `-n`? This would also be a breaking change requiring a major version bump.